packages/runtime-core/src/renderer.ts

```diff
@@ -1235,6 +1235,7 @@ function baseCreateRenderer(
       if (!initialVNode.el) {
         const placeholder = (instance.subTree = createVNode(Comment))
         processCommentNode(null, placeholder, container!, anchor)
+        initialVNode.el = placeholder.el
```

Comment:
this line also fixes #6463

Reply:
@edison1105 I updated the pull request description and title because it also address and add test for suspense runtime crashed now.